docs/manage/data/label.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Label Data"
+linkTitle: "Label Data"
+description: "Label image data on the DATA page in the Viam Cloud."
+weight: 39
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+images: [ "/manage/data/img/label-dog.gif" ]
+# SME: Tahiya Salam and Alexis Wei
+---
+
+To label data in a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **Images** subtab, you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+
+{{< alert title="Note" color="note" >}}
+Datasets are views, not materialized.
+If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
+
+Therefore the data you are viewing may change as you label and train on the dataset.
```

Comment:
```suggestion
The data you are viewing may change as you label and train on the dataset.
If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
```
[opt] As a layperson, I didn't know what "materialized" means in this context. If that's a standard way to put it, for sure leave it in. But otherwise maybe this is clearer?